gapic/src/main/com/google/gapid/perfetto/Unit.java

```diff
@@ -106,6 +108,21 @@ public String format(long value) {
       public String format(double value) {
         return numer.format(value) + "/" + denom.name;
       }
+
+      @Override
+      public Formatter withFixedScale(double representativeValue) {
+        return new Formatter() {
+          @Override
+          public String format(long value) {
+            return numer.withFixedScale(representativeValue).format(value) + "/" + denom.name;
```

Comment:
Lift `numer.withFixedScale` out of the anonymous class, so you have to compute it only once. i.e. something like
```
Formatter fixed = numer.withFixedScale(representativeValue);
return new Formatter() {
  // ...
 return fixed.format(value) + "/" + denom.name;
 // ...
}
```

Reply:
Sure, just did with a new commit.